### PR TITLE
chore(test): test

### PR DESCRIPTION
**Pull request by stavares843**
### What this PR does 📖
- 

### Which issue(s) this PR fixes 🔨
- Resolve #

### CODE REVIEW

**Comment by stavares843** [comment]
./build_sign_macos

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Rebase
+        run: |
+          # Check if the comment contains the `/rebase` command
+          if [[ "$GITHUB_EVENT_PATH" == *"/rebase"* ]]; then
+            # Rebase the pull request
+            git rebase origin/main
```
Comment (WanderingHogan) [inline .github/workflows/test.yml]: shouldn't this be dev?

**Review by WanderingHogan — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (stavares843) [reply .github/workflows/test.yml]: either works for the experiment in this case but updated 🔨

**Pull request merged**